Classes/Models/RepositoryLabel.swift

```diff
@@ -12,11 +12,11 @@ import IGListKit
 final class RepositoryLabel: ListDiffable, Hashable, Equatable, ListSwiftDiffable {
 
     let color: String
-    let name: String
+    var name: String = ""
```

Comment:
Any reason you changed this?

Reply:
If let you can’t mutate. Meaning I can’t switch `:bug:` no?

Reply:
You're only setting it once in the initializer though, isn't it?

Reply:
Oh yes! I originally had a separate fix (I created my
Own method that did the mapping. And I needed `self` but that doesn’t apply anymore. I’ll PR a quick fix to this soon. Thank you for pointing that out